lighthouse-core/lib/sd-validation/json-linter.js

```diff
@@ -9,21 +9,21 @@ const jsonlint = require('jsonlint-mod');
 
 /**
  * @param {string} input
- * @returns {{message: string, lineNumber: string|null}|null}
+ * @returns {{message: string, lineNumber: number|null}|null}
  */
 module.exports = function parseJSON(input) {
   try {
     jsonlint.parse(input);
   } catch (error) {
-    /** @type {string|null} */
+    /** @type {number|null} */
     let line = error.at;
```

Comment:
oh is `error.at` actually a number then?

Reply:
Yeah https://github.com/circlecell/jsonlint-mod/blob/master/lib/doug-json-parse.js#L67